apple/internal/apple_framework_import.bzl

```diff
@@ -223,6 +236,17 @@ def _apple_static_framework_import_impl(ctx):
     if ctx.attr.weak_sdk_frameworks:
         objc_provider_fields["weak_sdk_framework"] = depset(ctx.attr.weak_sdk_frameworks)
 
+    imported_swiftmodules = [
+        f
+        for f in header_imports
+        if _is_swiftmodule(f.basename)
+    ]
```

Comment:
even tho it's a bit more imperative it might be nice to build this list in the for loop above instead

Reply:
Do you mean in `_classify_framework_imports()`?

Reply:
ah nvm, better to leave here

Reply:
Actually, it's being removed per @sergiocampama's comment.